whatiusedtodo.html

```diff
@@ -29,14 +34,14 @@ <h1 align='center'> What I Used to Do </h1>
 
 <div class='container'>
 	<div class='row'>
-		<div class='col-sm-6' style="text-align:center">
+		<div class='col-sm-6 col-xs-12' style="text-align:center">
 			<h3 align='center'> What Jobs Have I Done in the Past?</h3>
-				<ul> Worked as a teaching assistant in a school in Vienna for 8 months </ul>
-				<ul> Worked as a catering assistant at the London 2012 Olympics working with fish and chips, waffles and beer!</ul>
-				<ul> Mowed the neighbour's farm lawns every weekend for years on a sit-on lawnmower - I miss this job and the countryside! </ul> <hr> <br>
+			<ul> Worked as a teaching assistant in a school in Vienna for 8 months </ul>
```

Comment:
`<ul>` is the unordered list container for `<li>` or list items, so this should be more like:

```
<ul>
  <li>Job one</li>
  <li>Job two</li>
  ...
</ul>
```